CatalogAPI/GetFilteredCatalogForPoint.py

```diff
@@ -0,0 +1,17 @@
+import requests
+
+lat = 55.862261
+lng = -4.250959
+zoom = 16
+token = "ADD YOUR TOKEN HERE"
+blackAndWhiteAllowed = False
+cloudCover = 20
+startDate = 1535371200000
+providers = "BLUESKY,DIGITALGLOBE"
+
+url = ("https://catalog-api.hibirdi.com/catalog/lat/{}/lng/{}/zoom/{}/token/{}?blackAndWhiteAllowed={}&cloudCover={}&startDate={}&providers={}"
+       .format(lat, lng, zoom, token, blackAndWhiteAllowed, cloudCover, startDate, providers))
```

Comment:
Maybe I missed a beat here but don't you want to include `colourAllowed` as well?

Reply:
I figured not; they'll both be in the API docs, but since they're both true by default, I thought it made sense for the example to show the intended usage of just turning one of them off